program-runtime/src/invoke_context.rs

```diff
@@ -108,6 +108,67 @@ impl ComputeMeter {
     }
 }
 
+// bprumo TODO: doc where 128 GB came from. Move where the const is, also
+// 128 GB
+pub const DEFAULT_MAX_ACCOUNTS_DATA_LEN: usize = 128_000_000_000;
+
+/// bprumo TODO: doc, and/or move
+#[derive(Debug, Default, Clone, Copy, Eq, PartialEq)]
+pub struct AccountsDataBudget {
+    /// The global maximum size for accounts data (i.e. DEFAULT_MAX_ACCOUNTS_DATA_LEN)
+    maximum: usize,
+    /// The current accounts data size (for a Bank)
+    current: usize,
+}
+impl AccountsDataBudget {
+    pub fn new(maximum: usize, current: usize) -> Self {
+        Self { maximum, current }
+    }
+    pub fn maximum(&self) -> usize {
+        self.maximum
+    }
+    pub fn current(&self) -> usize {
+        self.current
+    }
+    pub fn remaining(&self) -> usize {
+        self.maximum().saturating_sub(self.current())
+    }
+}
+
+/// bprumo TODO: doc, and/or move
+#[derive(Debug, Default, Clone, Copy, Eq, PartialEq)]
+pub struct AccountsDataMeter {
```

Comment:
Probably just need this, don't need the AccountDataBudget

Reply:
Yah. At the moment `AccountsDataBudget` is not really used for anything.